questionpy_server/settings.py

```diff
@@ -78,6 +82,49 @@ def transform_to_path(cls, value: Optional[DirectoryPath]) -> Optional[Directory
             return None
         return value.resolve()
 
+    @validator('repository_default_interval')
+    # pylint: disable=no-self-argument
+    def check_is_bigger_than_minimum_interval(cls, value: timedelta, values: dict[str, Any]) -> timedelta:
+        if value < values['repository_minimum_interval']:
+            raise ValueError(f"must be at least {values['repository_minimum_interval']}")
+        return value
+
+    @validator('repositories', pre=True)
+    # pylint: disable=no-self-argument
+    def transform_to_set_of_repositories(cls, value: str, values: dict[str, Any]) -> dict[str, Union[str, timedelta]]:
+        repositories: dict[str, Union[str, timedelta]] = {}
+
+        for line in value.splitlines():
+            if not line:
+                continue
+
+            # Split line into url and custom update interval.
+            url, _, custom_interval = line.partition(' ')
```

Comment:
Das geht kaputt, wenn Tabs oder mehrere Spaces verwendet werden, oder?

Lieber:
```suggestion
            # Split line into url and custom update interval.
            url, custom_interval = line.split(maxsplit=1)
```

Reply:
Tatsächlich, ich dachte `pydantic` kommt mit den zusätzlichen Spaces zurecht.